runtime/makelib/targets.mk.zos.inc.ftl

```diff
@@ -126,7 +126,17 @@ endif
 MRABIG = -Wc,"TBYDBG(-qdebug=MRABIG)"
 SPECIALCXXFLAGS = $(filter-out -Wc$(COMMA)debug -O3,$(CXXFLAGS))
 NEW_OPTIMIZATION_FLAG = -O2 -Wc,"TBYDBG(-qdebug=lincomm:ptranl:tfbagg)" -Wc,"FEDBG(-qxflag=InlineDespiteVolatileInArgs)"
-BytecodeInterpreter.o : BytecodeInterpreter.cpp
+
+BytecodeInterpreterFull.o : BytecodeInterpreterFull.cpp
+	$(CXX) $(SPECIALCXXFLAGS) $(MRABIG) $(NEW_OPTIMIZATION_FLAG) -c $< > $*.asmlist
+
+BytecodeInterpreterCompressed.o : BytecodeInterpreterCompressed.cpp
+	$(CXX) $(SPECIALCXXFLAGS) $(MRABIG) $(NEW_OPTIMIZATION_FLAG) -c $< > $*.asmlist
+
+BytecodeInterpreterFull.o : BytecodeInterpreterFull.cpp
+	$(CXX) $(SPECIALCXXFLAGS) $(MRABIG) $(NEW_OPTIMIZATION_FLAG) -c $< > $*.asmlist
+
+BytecodeInterpreterCompressed.o : BytecodeInterpreterCompressed.cpp
```

Comment:
There seems to be multiple entries for each interpreter in the makefiles.

Reply:
fixed